pyemma/coordinates/api.py

```diff
@@ -1414,9 +1415,113 @@ def vamp(data=None, lag=10, dim=None, scaling=None, right=True, ncov_max=float('
     return res
 
 
+def nystroem_tica(data, lag, max_columns,
```

Comment:
Please rename to ''tica_nystroem'' (makes it easier to find)

Reply:
OK, done.